pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -921,8 +922,13 @@ func translateFault(in *networking.HTTPFaultInjection) *xdshttpfault.HTTPFault {
 			out.Abort.ErrorType = &xdshttpfault.FaultAbort_HttpStatus{
 				HttpStatus: uint32(a.HttpStatus),
 			}
+		case *networking.HTTPFaultInjection_Abort_GrpcStatus:
+			code := grpcutil.SupportedGRPCStatus[a.GrpcStatus]
```

Comment:
@ckcd What if an invalid gRPC status code is passed? Is the status code validated somewhere before the lookup?

Reply:
Yes it's in `pkg/config/validation/validation.go` in this PR.